components/eventing-controller/README.md

```diff
@@ -55,14 +41,14 @@ This section explains how to use the Eventing Controller. It expects the followi
     | WEBHOOK_TOKEN_ENDPOINT | The Kyma public endpoint to provide Access Tokens.                              | BEB     |
     | DOMAIN                 | The Kyma cluster public domain.                                                 | BEB     |
```

Comment:
I think we need to document the env vars mentioned [for backend-controller](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/backend_config.go#L13-L27) as well.